miden/tests/integration/cli/cli_test.rs

```diff
@@ -38,3 +42,64 @@ fn cli_run() -> Result<(), Box<dyn std::error::Error>> {
 
     Ok(())
 }
+
+use assembly::{mast::MastNode, Library};
+
+#[test]
+fn cli_bundle_debug() {
+    let mut cmd = bin_under_test().command();
+    cmd.arg("bundle").arg("--debug").arg("./tests/integration/cli/data/lib");
+    cmd.assert().success();
+
+    let lib = Library::deserialize_from_file("./tests/integration/cli/data/out.masl").unwrap();
+    let found_one_decorator = lib.mast_forest().nodes().iter().any(|node| match node {
+        MastNode::Block(node) => !node.decorators().is_empty(),
+        _ => false,
+    });
```

Comment:
Decorators can be found in a basic block, but also as "pre-decorators" or "post-decorators" of all other node types (e.g. [here](https://github.com/0xPolygonMiden/miden-vm/blob/b7ff6c842b64392738019eb49f2a8c99d6557568/core/src/mast/node/join_node.rs#L22-L23)).

So what we really want to do is to iterate directly over `MastForest.decorators` (which might require adding a method to `MastForest`), and look for just one decorator with variant `Decorator::AsmOp` (unlike here where we look for any decorator variant).

Reply:
Done.